src/core/style/mixins.scss

```diff
@@ -326,3 +326,47 @@
     }
   }
 }
+
+// Only use when in row layout
+@mixin when-layout-row($element, $parent: '') {
```

Comment:
I've tried that, but the problem is that @media needs to be root level.

If I do ```
md-radio-group {
  @include when-layout-row(md-radio-button) {
    ...
  }
}```

Then the @media queries would appear under `md-radio-group`, I wasn't sure of a better solution.